app/assets/javascripts/atomic_cms.js

```diff
@@ -3,7 +3,7 @@
 
 (function() {
   'use strict';
-  var findElementIndex, page, positionEditBox, scrollTo, cmsType;
```

Comment:
yeah, dropping the cmsType function will break the ability to use this on models other than `Page`